src/main/java/blue/lapis/pore/impl/PoreServer.java

```diff
@@ -440,7 +464,13 @@ public void savePlayers() {
 
     @Override
     public boolean dispatchCommand(CommandSender sender, String commandLine) throws CommandException {
-        throw new NotImplementedException();
+        Preconditions.checkNotNull(sender, "sender");
+        Preconditions.checkNotNull(commandLine, "commandLine");
+
+        if (commandLine.startsWith("/")) {
```

Comment:
CraftBukkit doesn't do that (see https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/CraftServer.java#621). We're trying to emulate a CraftBukkit environment as good as possible, as this is were most plugins and users are coming from. Therefore this should match CraftBukkit's behaviour. Also, Glowstone doesn't seem to do this? https://github.com/GlowstoneMC/Glowstone/blob/master/src/main/java/net/glowstone/GlowServer.java#L1053-L1066